content/pt-br/docs/setup/production-environment/tools/kops.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Instalando Kubernetes com kOps
+content_type: task
+weight: 20
+---
+
+<!-- overview -->
+
+Este início rápido mostra como instalar facilmente um cluster Kubernetes na AWS usando uma ferramenta chamada [`kOps`](https://github.com/kubernetes/kops). 
+
+`kOps` é um sistema de provisionamento automatizado:
+
+* Instalação totalmente automatizada
+* Usa DNS para identificar clusters
+* Auto-recuperação: tudo é executado em grupos de Auto-Scaling
+* Suporte de vários sistemas operacionais (Amazon Linux, Debian, Flatcar, RHEL, Rocky e Ubuntu) - veja em [imagens](https://github.com/kubernetes/kops/blob/master/docs/operations/images.md)
+* Suporte a alta disponibilidade - consulte a [documentação sobre alta disponibilidade](https://github.com/kubernetes/kops/blob/master/docs/operations/high_availability.md)
+* Pode provisionar diretamente ou gerar manifestos do terraform - veja o [veja a documentação sobre como fazer isso com Terraform](https://github.com/kubernetes/kops/blob/master/docs/terraform.md)
```

Comment:
Minor: double word "veja o [veja a documentação ..."

Reply:
Fixed it